collections/made-in-switzerland/index.md

```diff
@@ -1,11 +1,14 @@
 ---
 items:
+ - SchweizerischeBundesbahnen/springboot-graceful-shutdown
+ - Sigmale1000/neonote
```

Comment:
Sorry, we've got a rule against adding your own repos 🙈

Reply:
Hey, thanks for the info! :)